projects/plugins/jetpack/changelog/add-domain-check-before-user-link-redirection

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: bugfix
+
+Fixed a problem with white screens or wrong redirections on the newsletter links.
```

Comment:
I think we could consider adding more specific information, e.g.:

```suggestion
User Content Link Tracking: check domain before redirecting to subscribe.wordpress.com
```

(taken from the related WPCOM patch)

Reply:
Thanks for the suggestions! Accepted both (: